cyclops-ui/src/components/pages/TemplateStore/TemplateStore.tsx

```diff
@@ -86,6 +91,28 @@ const TemplateStore = () => {
       });
   };
 
+  const checkTemplateReference = (repo: any, path: any, templateName: any) => {
+    setLoadingTemplateName(templateName);
+    axios
+      .get(`/api/templates?repo=${repo}&path=${path}`)
+      .then((res) => {
+        setLoadingTemplateName("");
+        setRequestStatus((prevStatus) => ({
+          ...prevStatus,
+          [templateName]: "success",
+        }));
+        message.success("Template reference is valid!");
```

Comment:
Can you also set an empty error when it succeeds? `setError({message: "", description: ""});`

Reply:
yeah sure!

Reply:
> Can you also set an empty error when it succeeds? `setError({message: "", description: ""});`

could you please explain, why are we adding an additional error handling here, since we are already handling the error in the catch block, I'm just curios😅, lookin forward to learn!!

Reply:
this is just an empty error that will close the error notification. Need to rewrite this to null or something, but we can do it in a separate PR

Reply:
@petar-cvit thank you! for the explanation and if you want I'll open another to rewrite it to null.